src/Jank/TermManagement/TermList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Term from './Term';
+import useApi from '../../hooks/useApi';
+import { AddButton, AddLabel, TermInput } from './ui';
+
+type Terms = {
+  _id: string,
+  value: string
+}
+
+const TermList = () => {
+  const [error, terms] = useApi<Terms[]>('/jank/terms');
+
+  if (error) {
+    return <span>Begriffe konnten leider nicht geladen werden</span>;
+  }
+  return (
+    <>
+      <h1>Begriffe verwalten</h1>
+      <form>
+        <AddLabel>
+          Begriff hinzufügen:
+          <TermInput type="text" name="word" />
```

Comment:
I guess `"word"` should be `"term"` or `"value"`, given that it doesn't have to be a single word and we use the word `term` in other contexts for the same thing.